frontend/packages/dev-console/src/components/add/AddCardSectionSkeleton.scss

```
@@ -0,0 +1,35 @@
+.odc-add-section-skeleton {
+  position: relative;
+  width: 100%;
+  &__background {
+    z-index: 1;
+    background-color: white;
+  }
+}
+
+.odc-add-section-skeleton-placeholder {
+  &__container {
+    position: absolute;
+    top: 0px;
```

nit: can avoid px with `0` , here and other places
```suggestion
    top: 0;
```